elements/map/src/plugins/advancedLayersAndSources/layers.ts

```diff
@@ -0,0 +1,11 @@
+import * as olLayers from "ol/layer";
+import STAC from "ol-stac";
+import { register } from "ol/proj/proj4.js";
+import proj4 from "proj4";
+
+register(proj4); // required to support source reprojection
+
+window.eoxMapAdvancedOlLayers = {
+  ...olLayers,
+  STAC,
```

Comment:
The README currently states "In order to use all formats, layers and sources provided by OpenLayers, import the plugin as well". Since STAC is a plugin of it's own (and not provided directly by `ol`), it should also be documented that it is included in our advanced plugin.

Reply:
Thank you, added to description!